test/fhirtypes/StructureDefinition.test.ts

```diff
@@ -97,11 +89,130 @@ describe('StructureDefinition', () => {
     });
   });
 
+  // Should get top level for '' element
   describe('#findElementByPath', () => {
-    it('should find an element by path', () => {
-      const valueX = observation.findElementByPath('value[x]');
+    let jsonRespRate: any;
+    let respRate: StructureDefinition;
+    beforeAll(() => {
+      jsonRespRate = defs.findResource('resprate');
```

Comment:
Isn't `resprate` the weird one that we said was wrong because it doesn't use slicing for the choice type?  That seems a little bit of an odd choice to use in a test unless you're purposefully testing to see if you can handle incorrect inputs.

Reply:
So `resprate` itself isn't weird, all of the vital sign profiles in this bundle we have use the same convention. Which is that instead of slicing `value[x]`, they directly create elements like `value[x].value` to represent `value[x]:valueQuantity.value`. Which seems to me to just be incorrect. I think once some of the weird behavior we were observing on the FHIR spec site is done, we may want to update this. But, I am not using `resprate` to test choice types. I only used `resprate` to test slicing, and to test elements that have children. The handling of those cases is correct.

Reply:
Actually, the body-weight profile (which is also a vital sign) seems to do it right:
http://hl7.org/fhir/R4/bodyweight.profile.json.html

Reply:
Ah.  Perhaps.  I thought I remembered looking at it before 4.0.1 and body-weight was right then too, but who knows.

We should update the FHIR definitions w/ the new 4.0.1 ones -- but I want to wait until we get the "all clear" to do so, as it seems they're not quite done yet.  Just this morning Grahame made a change to the 4.0.1 package.json file.